### PR TITLE
Vitalii logout

### PR DESCRIPTION
**Pull request by blishchenko1981**
Hi can I merge it"?

### CODE REVIEW

**Comment by blishchenko1981** [comment]
Ok, can someone check my branch?

**Comment by Sabirindev** [comment]
cool code

**Comment by blishchenko1981** [comment]
Thanks can u approve it please?

**Comment by AbbosB12** [comment]
Why I can't not see your merge on IntelliJ?

**Comment by AbbosB12** [comment]
Did you see my merge on IntellIJ?
Login Functionality

**Comment by blishchenko1981** [comment]
yes I do

**Comment by blishchenko1981** [comment]
I already aprove it from InteliJ

**Comment by AbbosB12** [comment]
Now, I see you merge on IntellIJ.
I refresh my page and see it.

**Comment by blishchenko1981** [comment]
Did u update project on InteliJ?

**Comment by blishchenko1981** [comment]
Ok cool.  Can U Approve mine merge?

**Comment by AbbosB12** [comment]
No. I refresh on page

**Comment by Sabirindev** [comment]
let me update my project through local master

**Review by AbbosB12 — CHANGES_REQUESTED** [review]
Can you added one more print statement, please?

**Comment by AbbosB12** [comment]
This code is very to merge?

**Comment by SHOKHZODJURAEV** [comment]
guys someone see my code?

**Review by SHOKHZODJURAEV — APPROVED** [review]

**Comment by SHOKHZODJURAEV** [comment]
guy are you here?

**Comment by SHOKHZODJURAEV** [comment]
hello

File: src/day4/Logout.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package day4;
+
+public class Logout {
+
+// work is done
+
```
Comment (Sabirindev) [inline src/day4/Logout.java:6]: what is the work you have done?

**Review by Sabirindev — APPROVED** [review]

**Pull request merged**